### ISSUE TITLE
nicegui ui.dialog slow/not smooth on Firefox on Android 

### ISSUE THREAD

**Issue by tidalvirus** [zauberzeug/nicegui#2334]
### Description

I'm using the very basic https://nicegui.io/documentation/dialog#dialog as my test, and a 'user experience' result as my indicator, compared against Chrome on the same Android tablets.

```python
from nicegui import ui

with ui.dialog() as dialog, ui.card():
    ui.label('Hello world!')
    ui.button('Close', on_click=dialog.close)

ui.button('Open a dialog', on_click=dialog.open)

ui.run()
```

My tablets are old - a Samsung Tab S4 (2018?), and Samsung Tab A (2017), and I thought they were just slow. But I accidentally browsed using Chrome instead of Firefox, and noticed that the experience of my app was a lot smoother in Chrome, so I tried it on both tablets, and it turns out the nicegui 'ui.dialog' (at least) is just slow in Firefox on Android,

I'm using Firefox 121.0.1 on both tablets, Chrome 120.0.6099.193 on an Android 10 (Tab S4), and Chrome 119.0.6045.193 on Android 7.1.1

I'm using nicegui on a linux host, python 3.11.2, nicegui 1.4.9. I noticed the same behaviour on nicegui 1.4.8.

Is this perhaps a known behaviour? Firefox on iOS doesn't have the same issue, but I understand that Firefox on iOS uses the same engine as every other browser.

TL;DR - ui.dialog() opens jerkily and slowly on Firefox on Android, but is super smooth/slick if using Chrome on the same tablet. (It might be more than just ui.dialog)

**Comment by rodja** [comment]
It would be interesting if a plain Quasar app with a dialog is also sluggy with Firefox on Android. Could you try?

**Comment by tidalvirus** [comment]
Ooh, good idea. I'm not sure how to do that (I used nicegui because I wanted to avoid /any/ sort of HTML/JS/CSS stuff 'manually') - if it helps, I'm using nicegui for a pretty simple app:  https://github.com/tidalvirus/grocy-family-assistant (just to show my level of knowledge here!)

I'll investigate a basic Quasar app tonight :)

**Comment by falkoschindler** [comment]
Here is a Quasar example with a dialog:
```html
<!DOCTYPE html>
<html>
  <head>
    <link href="https://cdn.jsdelivr.net/npm/quasar@2.14.2/dist/quasar.prod.css" rel="stylesheet" type="text/css" />
  </head>

  <body>
    <div id="q-app">
      <q-btn label="Alert" color="primary" @click="alert = true" />
      <q-dialog v-model="alert">
        <q-card>
          <q-card-section><div class="text-h6">Alert</div></q-card-section>
          <q-card-actions align="right"><q-btn flat label="OK" color="primary" v-close-popup /></q-card-actions>
        </q-card>
      </q-dialog>
    </div>

    <script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/quasar@2.14.2/dist/quasar.umd.prod.js"></script>

    <script>
      const app = Vue.createApp({
        setup() {
          return {
            alert: Vue.ref(false),
          };
        },
      });
      app.use(Quasar);
      app.mount("#q-app");
    </script>
  </body>
</html>
```

I basically took the code from https://quasar.dev/start/umd#installation and added a dialog from https://quasar.dev/vue-components/dialog.

**Comment by tidalvirus** [comment]
So - I made a simple quasar test app too, with a dialog, and it is indeed slow on Firefox on Android. I was then able to replicate by going to the Quasar doc site - which I wish I had thought of first :) https://quasar.dev/quasar-plugins/dialog/#dialog-api - the dialogs on there are slow on FF on Android too. I'll open a bug report on the Quasar side of things. Hopefully they'll know what the cause might be!
